extensions/codeforces-extension/package.json

```
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "codeforces-extension",
+  "title": "Codeforces",
+  "description": "Gives details about latest contests and details about any handle",
+  "icon": "command-icon.png",
+  "author": "harsh_varshney",
+  "categories": [
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-handle",
+      "title": "Search Handle",
+      "subtitle": "Codeforces",
+      "description": "Provides details about a handle",
+      "mode": "view"
+    },
+    {
+      "name": "search-contest",
+      "title": "Search Contest",
+      "subtitle": "Codeforces",
+      "description": "Lists upcoming and completed contests",
+      "mode": "view"
```

```suggestion
      "mode": "view",
      "arguments": [
        {
          "name": "handle",
          "placeholder": "Handle...",
          "type": "text",
          "required": true
        }
      ]
```

I would use an argument instead so you can save one view using [arguments](https://developers.raycast.com/information/lifecycle/arguments) - could you look into that?